GrowthBook/FeatureResult.cs

```diff
@@ -8,6 +9,17 @@ namespace GrowthBook
     [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
     public class FeatureResult
     {
+        /// <summary>
+        /// Represents possible values for the source of a feature result.
+        /// </summary>
+        public static class SourceIs
```

Comment:
I would recommend removing since it isn't used.
Also, I think you meant "SourceIDs"

Reply:
It's actually used when calling GetFeatureResult and was intended to be a more fluent-property kind of thing so it reads FeatureResult.SourceIs.DefaultValue or similar. Do you feel that it's more confusing that way?

Reply:
Actually, the more I think about it, we don't currently have much of a fluent-syntax usage in this, would add to the confusion. Renamed.